oauth2-useragent-core/src/main/java/com/microsoft/alm/oauth2/useragent/UserAgentImpl.java

```diff
@@ -110,6 +108,18 @@ AuthorizationResponse encode(final String methodName, final String... parameters
         catch (final InterruptedException e) {
             throw new AuthorizationException("interrupted_exception", e.getMessage(), null, e);
         }
+        catch (final AuthorizationException e) {
+            if ("parsing_error".equals(e.getCode()) && coordinator != null) {

```

Comment:
Can you please elaborate a little bit more on this?  

What I tried to do is there are two types of AuthorizationException that could throw from parsing the output now:
1. Parsing errors, somehow the stdout isn't really a name value pair
2. Legit server errors

So I don't want to hide the errorDescription if it's the second type.  Are you suggesting we could append to the errorContents to the errorDescription?